src/store/modules/tabs/reducer.ts

```diff
@@ -0,0 +1,34 @@
+import { Reducer } from 'redux';
+
+import { TabState, Action } from './types';
+import initialState from './state';
+
+const reducer: Reducer<TabState> = (state = initialState, action: Action) => {
```

Comment:
does the name of the reducer need to change to tabReducer (like in your other pr)?

Reply:
True